runtime/src/stakes.rs

```diff
@@ -253,31 +274,13 @@ impl Stakes<StakeAccount> {
                 return Err(Error::VoteAccountMismatch(*pubkey));
             }
         }
-        // Assert that all valid vote-accounts referenced in
-        // stake delegations are already cached.
-        let voter_pubkeys: HashSet<Pubkey> = stakes
```

Comment:
The old code was only processing *unique* vote accounts.
There are a lot more stake accounts than vote accounts.

Reply:
Isn't the new code doing exactly the same?

Reply:
Update: *now* I have removed the uniqueness check